### DISCUSSION TITLE
Autoscrape next episode?

### DISCUSSION THREAD

**Discussion by potatowe** [Goldenfreddy0703/Otaku#319] [Ideas]
Scraping the next episode before the previous episode finishes so you don't have wait for it to scrape.

**Comment by Goldenfreddy0703** [comment]
You still may have to wait for it to scrape sadly but hey, we have a new playback option called Smart Play where when you click on an episode, you select source, and when it goes to next episode, it will scrape that same source but with a higher episode number. Hope this helps.
